wordpress-mobile/gutenberg-mobile
# Navigation icon in bottomsheet is too dark on Dark Mode

Fixes #3956 

### Related PRs

* https://github.com/WordPress/gutenberg/pull/34806

To test:

* Using the demo app, open the cog-icon settings of a paragraph block and navigate in the "Font Size" setting.
* Notice the "back button" on the bottomsheet nav to not be too dark.

PR submission checklist:
